app/app.go

```diff
@@ -486,6 +486,11 @@ func NewElysApp(
 		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
 	)
 
+	bankParamSpace := app.GetSubspace(banktypes.ModuleName)
+	if !bankParamSpace.HasKeyTable() {
+		bankParamSpace = bankParamSpace.WithKeyTable(banktypes.ParamKeyTable())
+	}
```

Comment:
@kenta-elys do you need this?

Reply:
@cosmic-vagabond my bad! It was added while i was trying to fix the binary issue. Let me get rid of it.

Reply:
@cosmic-vagabond Done, removed